src/test_utils.rs

```diff
@@ -166,13 +166,25 @@ pub fn all_dialects() -> TestedDialects {
             Box::new(HiveDialect {}),
             Box::new(RedshiftSqlDialect {}),
             Box::new(MySqlDialect {}),
+            Box::new(MySqlNoEscapeDialect {}),
             Box::new(BigQueryDialect {}),
             Box::new(SQLiteDialect {}),
         ],
         options: None,
     }
 }
 
+pub fn all_dialects_other_than_mysqlnoescape() -> TestedDialects {
```

Comment:
the functions, "XX_other_than_mysqlnoescape", including this are supposed to be used for tests regarding escape.